src/Sarif.FunctionalTests/SarifValidatorTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+
+using FluentAssertions;
+using Microsoft.CodeAnalysis.Sarif.Driver;
+using Microsoft.CodeAnalysis.Sarif.Validation;
+
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.Sarif
+{
+    public class SarifValidatorTests
+    {
+        public const string TestDirectory = "SarifConverterTestData";
+        public const string JsonSchemaFile = "Sarif.schema.json";
+
+        private readonly string _jsonSchemaFilePath;
+
+        public SarifValidatorTests()
+        {
+            _jsonSchemaFilePath = Path.Combine(Environment.CurrentDirectory, JsonSchemaFile);
+        }
+
+        [Theory(Skip = "https://github.com/Microsoft/sarif-sdk/issues/108")]
```

Comment:
We can't actually run this test yet (although it works properly if you do run it), because it fails, because 22 of the sample files are in fact invalid. Tracked as #108.